### DISCUSSION TITLE
Can I transfer saved places from maps.me?

### DISCUSSION THREAD

**Discussion by sirensynapse** [organicmaps/organicmaps#8957] [Q&A]
Maps.me just lost contact with all my downloaded maps and wants me to redownload them, to which I said, nay, I won't pay. So I discovered organicmaps, yay! 
Can I move my saved locations over? They took me a godawful long time to make.

**Comment by biodranik** [comment]
If you can open Maps.Me, go to bookmarks, select any bookmark list, and then export it. Then select Organic Maps in the list of offered apps to import the KML file.

**Reply by sirensynapse** [reply]
Thanks, but it doesn't see my maps, bookmarks, anything. So that won't work. The files are there; can I manually copy them over? Where EXACTLY would I put them in android? :)
Edit: new trouble. The files were on the phone yesterday, but I decided to move them from internal storage to the SD card. A maps.me folder appeared there, noticed today, but it is empty. The files are GONE. Any idea what happened to them? Could doing a scan dig them up, or am I screwed? What bullshit. Not in internal memory either.

**Reply by biodranik** [reply]
You can copy kmb files to any folder, and then use the Import button in Organic Maps and specify that folder. Recently the kmb file format of MapsMe has been changed, so if it doesn't work, you can either wait for the next Organic Maps update or send me your files at alex at organicmaps dot app to help with conversion.

**Reply by sirensynapse** [reply]
Thanks again, just edited my last reply...read above. ;)

**Reply by biodranik** [reply]
It's hard to say what happened after a manual tinkering with files. If you had log enabled in OM settings, then we could check it.

**Reply by sirensynapse** [reply]
I didn't manually do it, I just used the app settings to migrate them, as my internal storage was getting full. Weird. 
No idea about log stuff...if it isn't on by default, it's not on. I never touched it.

**Reply by sirensynapse** [reply]
Ah wait, I found them. They are still in internal storage. Maps.me has two folders now, one called pro. I'll try to find and move the bookmarks now. 
I tried importing the saved places file, but organicmaps says it's corrupt. So I guess I will be starting over again. ;) Thanks for the help and the wonderful app! I'll be donating!
